@jalovisko I was able to build with pyinstaller, however when I try to run the dxf2png I get the following error:
```python
Traceback (most recent call last):
  File "main.py", line 19, in <module>
ModuleNotFoundError: No module named 'cv2'
[153549] Failed to execute script main
```

--- a/README.md
+++ b/README.md
@@ -46,12 +46,6 @@ Both input and output files are set relative to the script's root directory, i.e
 * Install the requirements (PIP is required): `pip install -r requirements.txt`
 * Deactivate the environment when done: `deactivate`
 
-##### Setup with Anaconda
-This assumes that you have the Conda package installed.
-* Create a virtual environment: `conda env create --name cad-mixer -f environment.yml`
-* Activate the environment: `conda activate cad-mixer`
-* Deactivate the environment when done: `conda deactivate`
-
 #### Building the binary executable
 
 This assumes that the dependencies were installed with PIP in a virtual environment (see the **Setup with Virtualenv** section). Conda handles dependencies in a trickier way and this might not work.

--- a/README.md
+++ b/README.md
@@ -52,7 +52,7 @@ This assumes that the dependencies were installed with PIP in a virtual environm
 
 * Extra step for ArchLinux: PyInstaller doesn't work on ArchLinux as intended in some cases as mentioned in [this issue here](https://github.com/pyinstaller/pyinstaller/issues/5540). A workaround is provided in a bash script and can be ran simply as `chmod +x arch_pyinstaller.sh && bash arch_pyinstaller.sh`
 * Go to the dxf2png folder (if not there already): `cd dxf2png`
-* Run the installation: `pyinstaller --paths="./../cad-mixer/lib" -n dxf2png --exclude-module tkinter --onefile main.py`
+* Run the installation: `pyinstaller --paths="./../venv/lib" -n dxf2png --exclude-module tkinter --onefile main.py`
 
 The resulting executable file is placed to the `dist` folder at the same path as `main.py`. It can be used as follows:
 ```sh

--- a/README.md
+++ b/README.md
@@ -41,8 +41,8 @@ Both input and output files are set relative to the script's root directory, i.e
 #### Preparing the environment on Linux
 
 ##### Setup with Virtualenv
-* Create a virtual environment: `virtualenv -p /usr/bin/python3 cad-mixer`
-* Activate the environment: `source cad-mixer/bin/activate`
+* Create a virtual environment: `virtualenv -p /usr/bin/python3 venv`
+* Activate the environment: `source venv/bin/activate`
 * Install the requirements (PIP is required): `pip install -r requirements.txt`
 * Deactivate the environment when done: `deactivate`
 

